packages/Webkul/Core/src/Models/Channel.php

```diff
@@ -68,6 +69,14 @@ public function inventory_sources()
         return $this->belongsToMany(InventorySourceProxy::modelClass(), 'channel_inventory_sources');
     }
 
+    /**
+     * Get the product ordered inventory associated with channel.
+     */
+    public function product_ordered_inventories()
```

Comment:
Remove this two-way relationship from here as we don't want the Core package to dependent on other package